spring/spring-boot-4/embedded/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>infinispan-spring-boot4-starter-parent</artifactId>
+    <groupId>org.infinispan</groupId>
+    <version>16.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Infinispan Spring Boot 4 Starter Embedded</name>
+  <artifactId>infinispan-spring-boot4-starter-embedded</artifactId>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-configuration-processor</artifactId>
+      <optional>true</optional>
+    </dependency>
+
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-cache</artifactId>
+        <version>${version.spring.boot4}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
```

Comment:
The project doesn't have a starter that isn't optional. Shouldn't you have at least `spring-boot-starter`? 

As for actuator in optional scope, perhaps you could trim down to only metrics (if that's what you need from it). There is a `spring-boot-micrometer-metrics` that could be more scoped.